hw03_hilairek.py

```diff
@@ -0,0 +1,206 @@
+# Author: Kervens Hilaire
+# Username: hilairek
+#
+# Assignment: HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose: Learning more about functions, git, and RGB.
+# Google Doc Link: https://docs.google.com/document/d/1qX_R1QL3Qg3_BBGEiCuX8spbrTxbjD2ipZlsZpv2VWk/edit
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+import turtle
+
+
+def drawroof():
+    """
+    This function draws the roof
+    """
+
+    t = turtle.Turtle()
+    t.hideturtle()
+    t.speed(0)
+    t.penup()
+    t.goto(-200, 50)
+    t.pendown()
+    t.color('red')
+    t.begin_fill()
+    for i in range(1):
+        t.forward(450)
+        t.left(110)
+        t.forward(199)
+        t.left(70)
+        t.forward(300)
+        t.left(66)
+        t.forward(205)
+    t.end_fill()
+
+
+def drawroofwindow():
+    """
+    This function draws the roof window
+
+    """
+    t = turtle.Turtle()
+    t.hideturtle()
+    t.speed(0)
+    t.color("white")
+    t.penup()
+    t.goto(-60, 100)
+    t.pendown()
+    t.pensize(5)
+    t.forward(130)
+    for i in range(2):
+        t.left(90)
+        t.forward(80)
+        t.left(90)
+        t.forward(130)
+    t.backward(65)
+    t.left(90)
+    t.forward(80)
+    t.backward(40)
+    t.right(90)
+    t.forward(60)
+    t.backward(120)
+
+
+def bodyofhouse():
+    """
+    This function draws the body of a house
+    """
+    t = turtle.Turtle()
+    t.speed(0)
+    t.penup()
+    t.goto(-190, 50)
+    t.pendown()
+    t.hideturtle()
+    t.color("#d2b48c")
+    t.begin_fill()
+    for i in range(1):
+        t.forward(426)
+        t.right(90)
+        t.forward(256)
+        t.right(90)
+        t.forward(426)
+        t.right(90)
+        t.forward(256)
+        t.end_fill()
+
+
+def door():
+    """
+    This function draws the door of the house
+    """
+    t = turtle.Turtle()
+    t.speed(0)
+    t.penup()
+    t.goto(-160, -207)
+    t.pendown()
+    t.hideturtle()
+    t.color("#556b2f")
+    t.begin_fill()
+    t.left(90)
+    t.forward(150)
+    t.right(90)
+    t.forward(100)
+    t.right(90)
+    t.forward(150)
+    t.right(90)
+    t.forward(100)
+    t.end_fill()
+    t.penup()
+    t.backward(200)
+    t.right(90)
+    t.forward(110)
+    t.pendown()
+    t.pencolor("white")
+    t.pensize(3)
+    for i in range(1):
```

Comment:
This only runs once (therefore, it's not a loop). A loop helps us not have to write the same lines of code over and over again because an operation is repeating. Even with slight variances (like the `forward(100)` vs. `forward(150)`, you can handle these in a loop:

eg:
```
for distance in [150, 100]:
    t.forward(distance)
    t.right(90)
```